wagtail/users/tests/test_admin_views.py

```diff
@@ -227,6 +227,50 @@ def test_search_query_one_field(self):
         results = response.context["users"]
         self.assertIn(self.test_user, results)
 
+    @unittest.skipUnless(
+        settings.AUTH_USER_MODEL == "customuser.CustomUser",
+        "Only applicable to CustomUser",
+    )
+    @override_settings(
+        WAGTAIL_USER_CREATION_FORM="wagtail.users.tests.CustomUserCreationForm",
+        WAGTAIL_USER_CUSTOM_FIELDS=["country", "document"],
+    )
+    def test_search_query_one_searchable_field(self):
+        custom_user_with_country = self.create_user(
+            username="testjoe",
+            email="testjoe@email.com",
+            password="password",
+            first_name="Joe",
+            last_name="Doe",
+            country="testcountry",
+        )
+        response = self.get({"q": "country"})
```

Comment:
Even with an `autocomplete` search, a search term normally only matches the start of a word, so searching for "country" would not be valid - this (and the test below) should really be searching for something like "testco". (In fact, this test does currently pass, because the test suite only runs against the fallback database search backend, which does simple substring matches - but we're working towards fixing this to use the "real" search backends - see #10696).